private-templates-service/client/src/store/auth/actions.ts

```diff
@@ -0,0 +1,16 @@
+import { LOGIN, LOGOUT, UserType, AuthAction } from './types';
+
+export function login(user: UserType): AuthAction {
+  return {
+    type: LOGIN,
+    text: 'User login',
+    user: user
+  }
+}
+
+export function logout(): AuthAction {
+  return {
+    type: LOGOUT,
+    text: 'User Logout'
```

Comment:
Text on line 6 has a lowercase second letter and this one has uppercase, should change one of them so they are consistent